src/com/mendix/widget/carousel/components/__tests__/Carousel.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { ShallowWrapper, shallow } from "enzyme";
+import { DOM, createElement } from "react";
+
+import { Carousel, CarouselProps, Image } from "../Carousel";
+import { CarouselItem } from "../CarouselItem";
+
+describe("Carousel", () => {
+    let images: Image[];
+    let carousel: ShallowWrapper<CarouselProps, any>;
+    let carouselWrapper: ShallowWrapper<CarouselProps, any>;
+
+    it("renders the structure correctly", () => {
+        images = [ { url: "https://www.google.com/images/nav_logo242.png" } ];
+        carousel = shallow(createElement(Carousel, { images }));
+
+        expect(carousel).toBeElement(
+            DOM.div({ className: "mx-carousel" },
+                DOM.div({ className: "mx-carousel-item-wrapper" },
+                    createElement(CarouselItem, { active: true, url: images[0].url })
+            )));
```

Comment:
Indentation doesn't match the structure, the second `DOM.div` doesn't have a closing parentheses on the same indentation level.